atomata/capstone_WebClient
# Login page

### Summary 
Adds login page with Cookies to validate user's credentials
Credentials will determine whether a user should have access to a page or not
### How to test?
1. Start the project
2. You will automatically be re-directed to the new Login page
3. Try and access the Experience, Selection, and Index pages without logging in
4. By default, you should be re-directed back to the Login page every time
5. Login using any username (AT LEAST 3 CHARACTERS LONG) and password (Login page validates if both fields are empty)
6. Arrive to the Index page and try to get back to the Login page
7. This should return you to the Index page
8. Repeat this with the other pages and ensure all pages are still fully functional 
9. Return to Index and try to log out
10. Logging out should delete your cookies and return you to the Login page with no issues
11. Try and access the Experience, Selection, and Index pages again
